platform-sdk/swirlds-common/src/main/java/com/swirlds/common/metrics/RunningAverageMetric.java

```diff
@@ -100,11 +99,13 @@ default EnumSet<ValueType> getValueTypes() {
     final class Config extends MetricConfig<RunningAverageMetric, RunningAverageMetric.Config> {
 
         private final double halfLife;
+        private final boolean useDefaultHalfLife;
 
         /**
          * Constructor of {@code RunningAverageMetric.Config}
          *
-         * The {@code halfLife} is by default set to {@code SettingsCommon.halfLife}.
+         * The {@code useDefaultHalfLife} is set to {@code true} and have to check
```

Comment:
Better:
The {@code useDefaultHalfLife} defines if the default halfLife (see MetricsConfig#halfLife()) should be used when creating a metric based on this config or if the halfLife of the config should be used.